unfoldingWord/translationCore
# USFM3 Export

#### This pull request addresses:
Ability to export alignment data from tC WordAlignment tool

#### How to test this pull request:
Make sure you have the latest usfm-js branch: ```1.0.0-beta.21``` (at the moment)
If you don't just run ```npm i```
Start the app
Make sure you have a working project in my projects.
You can try a project with or without alignments.
On the projects screen click on the three dot button for project you want to export.
Click on ```Export Alignments``` option.
Once finished you should see an alert saying it saved.
Open it and ensure the metadata for the alignments matches the alignments you actually made on the app.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3747)
